<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewDayReserve</class>
 <widget class="QDialog" name="NewDayReserve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>554</height>
    </rect>
   </property>
   <property name="title">
    <string> Nova </string>
   </property>
   <widget class="QGroupBox" name="groupBoxHorarios">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>166</y>
      <width>511</width>
      <height>380</height>
     </rect>
    </property>
    <property name="title">
     <string> Horários</string>
    </property>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>331</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ContiguousSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QPushButton" name="pushButtonInvited">
     <property name="geometry">
      <rect>
       <x>373</x>
       <y>354</y>
       <width>129</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Convidados...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>48</y>
      <width>510</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Responsável</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="maxVisibleItems">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>82</y>
      <width>513</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Iniício</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEditStart">
       <property name="maximumTime">
        <time>
         <hour>23</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Fim</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="timeEditEnd"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Local</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLocal"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBoxRepeat">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>114</y>
      <width>510</width>
      <height>53</height>
     </rect>
    </property>
    <property name="title">
     <string> Repetir</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Forma</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxRepetirForma">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Diariamente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Semanalmente</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mensalmente</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Vezes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="spinBoxVezes">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonSave">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>571</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditLocal</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>timeEditStart</tabstop>
  <tabstop>timeEditEnd</tabstop>
  <tabstop>groupBoxRepeat</tabstop>
  <tabstop>comboBoxRepetirForma</tabstop>
  <tabstop>spinBoxVezes</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>pushButtonSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
  <include location="../../icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
